debezium-connector-mysql/src/test/java/io/debezium/connector/mysql/MySqlGeometryTest.java

```diff
@@ -7,28 +7,54 @@
 
 import org.junit.Test;
 
-import mil.nga.wkb.geom.Point;
-
 import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertTrue;
 
 /**
  * @author Omar Al-Safi
  */
 public class MySqlGeometryTest {
```

Comment:
Could you please xtend the test with all geom types supported? We need to make sure that MySQL+BinlogClient+value converter combo works well for all cases.

Reply:
Done